src/dialogs/config-flow/show-dialog-config-flow.ts

```diff
@@ -22,12 +23,14 @@ export const showConfigFlowDialog = (
   showFlowDialog(element, dialogParams, {
     loadDevicesAndAreas: true,
     getFlowHandlers: async (hass) => {
-      const [integrations, helpers] = await Promise.all([
+      const [integrations, helpers, supportedBrands] = await Promise.all([
         getConfigFlowHandlers(hass, "integration"),
         getConfigFlowHandlers(hass, "helper"),
+        getSupportedBrands(hass),
         hass.loadBackendTranslation("title", undefined, true),
       ]);
-      return { integrations, helpers };
+
+      return { integrations, helpers, supportedBrands };
```

Comment:
Changing the return value means you need to update the method signature of `getFlowHandlers` in the interface.